### DISCUSSION TITLE
Using peek for screen capture

### DISCUSSION THREAD

**Discussion by RaZ0rr-Two** [herbstluftwm/herbstluftwm#1379] [Q&A] [answered]
Is anyone using [peek](https://github.com/phw/peek)  with hlwm ?

I have picom which allows peek to capture screen but I cannot see inside.

If anyone has it set up, then please share it.

**Comment by t-wissmann** [comment, marked as answer]
Here, peek works as expected. What do you mean by "cannot see inside"? Here, I can see through the hole of the peek window as expected.

(By the way: if you want to *click* inside the peek window, you need the current git version of hlwm and need to disable the window decorations of the window)

**Comment by RaZ0rr-Two** [comment]
Oops. I'm really sorry. I can see inside because of picom. But I cannot click inside. I'll try the git version. But what do you mean by window decorations? Which setting is that referring to?

**Reply by t-wissmann** [reply]
Run:
```
hc keybind Mod4-Shift-d attr clients.focus.decorated toggle
```
and then while `peek` is focused, press <kbd>Mod4-Shift-d</kbd>

**Reply by RaZ0rr-Two** [reply]
Thanks for the help. Shall I create an issue for this so that others can find it?

**Reply by t-wissmann** [reply]
No, I think it will be solved automatically in upcoming releases. And for the time being, we have the present discussion. (Congratulations :tada: by the way for being the first person to use the discussion on the hlwm repo :tada: :smile_cat:  )

**Comment by RaZ0rr-Two** [comment]
Thanks. I thought this was more suited for the subreddit but I saw the discussion option.
